docs/basic-usage/working-with-the-current-tenant.md

```diff
@@ -13,12 +13,20 @@ You can find the current method like this.
 Spatie\Multitenancy\Models\Tenant::current(); // returns the current tenant, or if not tenant is current, `null`
 ```
 
-A current tenant will also be bound in the container using the `currentTenant` key.
+A current tenant will also be bound in the container using the `currentTenant` key and `Spatie\Multitenancy\Models\Contracts\CurrentTenant` class.
 
 ```php
 app('currentTenant'); // returns the current tenant, or if not tenant is current, `null`
 ```
 
+```php
+use Spatie\Multitenancy\Models\Contracts\CurrentTenant;
+
+Route::get('/current-tenant', function (CurrentTenant $currentTenant) {
```

Comment:
What happens if there is no current tenant?

Reply:
Good point, currently it'll new up a new `Tenant` instance but I'll change it to be `null`

Reply:
Hm, same issue exists for the `currentTenant`. I'll make both `null`